### ISSUE TITLE
Prevent "Interface Scale" editor option from applying to in-game UI.

### ISSUE THREAD

**Issue by Menotdan** [FlaxEngine/FlaxEngine#2347]
I think it would be a good idea to prevent the "Interface Scale" option from applying to Game UI while observing it in the editor, since the interface scale doesn't apply to a cooked game, and showing the large UI makes it hard to see what the UI would look like without cooking the game.

**Comment by Tryibion** [comment]
I can not seem to replicate this... Changing interface scale in my editor does not seem to affect the in game ui...

**Comment by Menotdan** [comment]
> I can not seem to replicate this... Changing interface scale in my editor does not seem to affect the in game ui...


https://github.com/FlaxEngine/FlaxEngine/assets/32620310/0b6c9aa7-5690-4e4b-ab97-e3c7e9e6fdb6

Here it is. Maybe it's because I don't have a canvas scaler? (I probably want one but thats irrelevant atm)

**Comment by Tryibion** [comment]
Ah, my bad. I was using a canvas scalar.

**Comment by mafiesto4** [comment]
Maybe it's a feature?

**Comment by Menotdan** [comment]
> Maybe it's a feature?

No, I find it quite annoying, and making it apply to the cooked game doesn't make sense imo since it's my *editor* settings. I just want my in-game UI to match in the editor and the cooked game.
